projects/commudle-admin/src/app/feature-modules/sys-admin/components/admin-badges/admin-badges-assign/admin-badges-assign.component.scss

```diff
@@ -0,0 +1,44 @@
+@import '/projects/commudle-admin/src/assets/styles/variables';
+
```

Comment:
Why have we imported the variables.scss if we are not using it?

Reply:
we have a use for it now as we are giving tab width from this file